apps/web-giddh/src/app/invoice/eWayBill/eWayBill/eWayBill.component.ts

```diff
@@ -232,8 +226,8 @@ export class EWayBillComponent implements OnInit, OnDestroy {
         });
         this.store.pipe(select(p => p.ewaybillstate.EwayBillList), takeUntil(this.destroyed$)).subscribe((o: IEwayBillAllList) => {
             if (o) {
-                this.EwaybillLists = _.cloneDeep(o);
-                this.EwaybillLists.results = o.results;
+                this.ewaybillLists = _.cloneDeep(o);
```

Comment:
```suggestion
                this.ewaybillLists = cloneDeep(o);
```
Changes in over all file